### ISSUE TITLE
Issue: The star went missing!

### ISSUE THREAD

**Issue by sapioit** [Greentwilight/VIPs#14]
Yeah, what the name says.

**Comment by LoganTCP** [comment]
Hi sapioit,

I found a workaround for now until the developer fixes the plugin.

**Step 1:** Enable Developer Mode

Navigate to: User Settings > Appearance > Advanced > Developer Mode > ON
![Vips-fix1](https://user-images.githubusercontent.com/34076740/62426024-c9a2d500-b6ad-11e9-8260-234932ed40b7.png)

**Step 2:** Copy User ID

Close your User Settings, now right-click a user you want to add to your VIP list.
In the context menu, click "Copy ID".
![Vips-fix2](https://user-images.githubusercontent.com/34076740/62426082-53eb3900-b6ae-11e9-95e2-5b3d3845c0a0.png)

**Step 3:** Open the config file

Press and hold the `Windows Key + R` keys on your keyboard.
In the text area type `%APPDATA%\BetterDiscord\plugins` and hit the `[RETURN]` key.
A Windows Explorer window will appear, look for "VIPs.config.json".
Edit the file with you favorite text editor. (In this tutorial I am using [Notepad++](https://notepad-plus-plus.org/))

**Step 4:** Editing the config file

Under `"ids": [` you will find numbers, these are your current VIPs'.
![Vips-fix3](https://user-images.githubusercontent.com/34076740/62426361-cc072e00-b6b1-11e9-8a68-d6fbd714d5f8.png)

To add a VIP add the User ID (from step 2) in the following format:
`"User ID HERE",`
`"Another User ID HERE",`
`"Last User ID HERE"`

Note: When you get to your last ID DO NOT ADD A COMMA thus causing the plugin to fail
Save your config file. (`Ctrl + S` on Notepad++)

**Step 5:** Refresh the plugin

There are 3 ways of achieving this.

**Way 1:** Refresh Discord by pressing `Ctrl + R`
**Way 2:** Close and Reopen Discord
**Way 3:** Disable and then re-enable the VIPs plugin

Hope this helps! Good Luck!!!

- LoganTCP / TallyMark

**Comment by sapioit** [comment]
Thank you! Much appreciated!

**Comment by sapioit** [comment]
By the way, if you have the _Search Everything_ (by Voidtools) app/software, finding the file is much easier.

**Comment by LoganTCP** [comment]
True, but when I respond to things I like to make it a bone stock as I can.

Glad I could help, have a great day!

**Comment by sapioit** [comment]
I understand. I was just giving a random tip.
 Thank you! Have a great day!
